docker-compose.yml

```diff
@@ -1,13 +1,35 @@
-version: '3'
+# version 3+ does different things re: networking from 2.x
+# mainly things to support swarms
+# aka things we don't need/want
+version: '2.4'
 services:
-
-  fishblog:
-    image: fishnet/blog
-    container_name: fishblog 
+  fishblog-app:
+    container_name: fishblog-app
+    # production: track "prod" tag on ghcr.io
+    # image: ghcr.io/wetfish/blog:prod
+    # for development, build the image
+    # TODO: once ghcr.io autobuilds in place, change this
+    build: .
+    
+    # restart unless we stopped it manually
     restart: unless-stopped
-    tty: true
-    working_dir: /app
+
+    # our config file
     volumes:
-      - ./:/app
-    ports:
-      - "2304:2304"
+      - "./config.js:/app/server/config.js:ro"
+
+    # uncomment to expose port directly
```

Comment:
I think it would be more developer friendly to include two docker-compose files. One for the production deployment and the other for local development

Reply:
How do the additions in [https://github.com/wetfish/blog/pull/34/commits/03e035bf716141766aa5c13dc988490c199881fd](https://github.com/wetfish/blog/pull/34/commits/03e035bf716141766aa5c13dc988490c199881fd) look?

Reply:
Looking great :tada: